Resources/Prototypes/Entities/Clothing/Head/hats.yml

```
@@ -273,6 +282,7 @@
   - type: Tag
     tags:
     - ClothMade
+        - Recyclable
```

```suggestion
    - ClothMade
    - Recyclable
```
Test fail real. Fix the intendation here.